flyers/fr/openSUSE-Cheat-Sheet-BG_fr.svg

```diff
@@ -0,0 +1,5589 @@
+     style="fill:#66bf2e;fill-opacity:1;fill-rule:nonzero;stroke:none" />
+  <path
+     inkscape:connector-curvature="0"
+     id="path233"
+     d="m 1181.3633,126.09375 1.2383,0 0.2109,-1.5 -1.2383,0 z m -1.7617,0.68359 0,-0.68359 1.0703,0 0.2109,-1.5 -1.0195,0 0,-0.67969 1.1094,0 0.289,-2.04687 0.6914,0 -0.289,2.04687 1.2382,0 0.2891,-2.04687 0.6914,0 -0.289,2.04687 0.8906,0 0,0.67969 -0.9805,0 -0.2109,1.5 0.9101,0 0,0.68359 -1.0117,0 -0.2891,2.0586 -0.6914,0 0.293,-2.0586 -1.2422,0 -0.289,2.0586 -0.6914,0 0.2929,-2.0586 z m 0,0"
+     style="fill:#66bf2e;fill-opacity:1;fill-rule:nonzero;stroke:none" />
+  <path
+     inkscape:connector-curvature="0"
+     id="path235"
+     d="m 1191.7227,128.83594 0,-0.82032 -0.02,0 c -0.1797,0.32032 -0.4101,0.55469 -0.6914,0.70313 -0.2773,0.15234 -0.6094,0.22656 -0.9883,0.22656 -0.3398,0 -0.625,-0.0469 -0.8515,-0.13672 -0.2266,-0.0898 -0.4102,-0.21875 -0.5508,-0.38281 -0.1367,-0.16797 -0.2383,-0.36719 -0.293,-0.59765 -0.059,-0.22657 -0.086,-0.48438 -0.086,-0.76172 l 0,-3.40235 0.8516,0 0,3.5 c 0,0.32032 0.09,0.57422 0.2773,0.76172 0.1875,0.18359 0.4453,0.27734 0.7695,0.27734 0.2617,0 0.4883,-0.0391 0.6758,-0.11718 0.1914,-0.082 0.3477,-0.19531 0.4766,-0.33985 0.125,-0.14843 0.2226,-0.32031 0.2851,-0.51562 0.062,-0.19922 0.094,-0.41406 0.094,-0.64453 l 0,-2.92188 0.8515,0 0,5.17188 z m 0,0"
+     style="fill:#66bf2e;fill-opacity:1;fill-rule:nonzero;stroke:none" />
+  <path
+     inkscape:connector-curvature="0"
+     id="path237"
+     d="m 1194.6016,123.66406 0,0.82032 0.02,0 c 0.3555,-0.625 0.9141,-0.94141 1.6797,-0.94141 0.3398,0 0
```

Comment:
YaST no longer has GTK UI option (since a few years back)

Reply:
Fixed, I got misleaded by the fact that the option "--gtk" still exists in `yast --help` but, indeed, falls back to "--qt".